shared/src/main/scala/amf/core/resolution/pipelines/ResolutionPipeline.scala

```diff
@@ -1,43 +1,20 @@
 package amf.core.resolution.pipelines
 
-import amf.ProfileName
 import amf.core.benchmark.ExecutionLog
 import amf.core.errorhandling.ErrorHandler
 import amf.core.model.document.BaseUnit
 import amf.core.resolution.stages.ResolutionStage
 
 import scala.scalajs.js.annotation.{JSExportAll, JSExportTopLevel}
 
-abstract class ResolutionPipeline(val eh: ErrorHandler) {
```

Comment:
Why still calling this ResolutionPipeline while renaming everything else to "transformation"?

Reply:
True, a full renaming of ResolutionPipeline to TransformationPipeline is pending